.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
```

Comment:
I agree that typed arrays have been around for a while (and in Cesium)...With somewhat dubious origins and support (and no mention in the ES5 spec).

After looking through some Cesium issues, I am now wondering if there was an un-guarded (no feature detection check) typed array usage added some time after v1.29.

In any case, I will use the eslint globals attribute to address the typed arrays in the code and avoid using `"es6": true` (and see if I can find what is causing an issue between PhantomJS and v1.33)